lib/bap_primus/bap_primus_lisp_resolve.ml

```diff
@@ -107,18 +108,24 @@ let overload_macro code (s3) =
   | ((n,_,_) as c) :: cs -> List.filter_map (c::cs) ~f:(fun (m,d,bs) ->
       Option.some_if (n = m) (d,bs))
 
-let all_bindings f = 
+let all_bindings f =
   List.for_all ~f:(fun (v,x) ->
       f v.data.typ x)
 
+let zip x y =
+  let open List.Or_unequal_lengths in
```

Comment:
There is no need for this open, as now the compiler will take the constructors from the right place.